systemtest/src/main/java/io/strimzi/systemtest/matchers/LogHasNoUnexpectedErrors.java

```diff
@@ -64,12 +64,13 @@ public void describeTo(Description description) {
                 + "(?s)(.*?)"
                 + "io.fabric8.kubernetes.client.KubernetesClientException: Failure executing: PATCH"),
         // fabric8 now throws exceptions, which doesn't influence kafka scaleup/scaledown
-        FABRIC_EIGHT_SCALEUP_ERROR("ERROR StatefulSetOperationsImpl:115 - Error while waiting for resource to be scaled."),
-        FABRIC_EIGHT_STATEFUL_SET_SCALEUP_ERROR("ERROR StatefulSetOperationsImpl:[0-9]+ - 0/.* pod(s) ready for StatefulSet: .*  after waiting for 0 seconds so giving up"),
+        FABRIC_EIGHT_SCALEUP_ERROR("ERROR StatefulSetOperationsImpl:[0-9]+ - Error while waiting for resource to be scaled."),
```

Comment:
FYI: This should be already fixed in master.

Reply:
I see, so I will check it later and remove in next PR, do you agree?

Reply:
Yeah, fine with me.